docs/pages/access-controls/access-requests/role-requests.mdx

```diff
@@ -260,7 +260,14 @@ $ tctl request approve --roles=role-1,role-3 --reason='Approved, but not role-2
   older versions of Teleport, since only the issuing Auth Server needs any particular
   knowledge of the feature.
 
+## Next Steps
 
-## Integrating with an External Tool
+### Integrating with an external tool
 (!docs/pages/includes/access-request-integrations.mdx!)
 
+### Using TTLs with Access Requests
+
+`tsh request create` supports flags to control TTLs for the request and
```

Comment:
```suggestion
Users can also create Access Requests with the `tsh request create` command.
`tsh request create` supports flags to control TTLs for the request and
```

Since we don't mention this command earlier in the guide